lib/compat/wordpress-6.1/template-parts-screen.php

```diff
@@ -24,6 +24,24 @@ function gutenberg_template_parts_screen_menu() {
 		return;
 	}
 
+	global $submenu;
+	if ( ! isset( $submenu['themes.php'] ) ) {
+		return;
+	}
+
+	$needs_custom_page = true;
+	foreach ( $submenu['themes.php'] as $menu_item ) {
+		if ( str_contains( $menu_item[2], 'site-editor.php?postType=wp_template_part' ) ) {
```

Comment:
I'm not sure whether there is any other way to query this, but just from my gut feeling this feels rather fragile. But I will still approve the PR since we can follow up with a more robust solution and this does fix the bug that is currently present.